### ISSUE TITLE
Revisión sitios 2020

### ISSUE THREAD

**Issue by n3storm** [python-spain/empresas#37]
- Openshine se despide en su propia web https://openshine.com
- Gisce.net tiene caducado el certificado SSL 25/12/2019 y no twittea nada. Contactados por twitter a ver si dan señales.
- pixelmouse.es tiene caducado el certificado 8/8/2020. Contactados por twitter a ver si dan señales.

**Comment by quobit** [comment]
¡Muchas gracias!
¿Cuánto sería un tiempo prudencial? /cc @astrojuanlu

**Comment by astrojuanlu** [comment]
@tanrax :)

yo creo que mejor no tener info desactualizada. propongo pinguear a quienes abrieran el PR original, y si en 1 mes no hay respuesta se borra

**Comment by quobit** [comment]
>     * Openshine se despide en su propia web https://openshine.com
> 
>     * Gisce.net tiene caducado el certificado SSL 25/12/2019 y no twittea nada. Contactados por twitter a ver si dan señales.
> 
>     * pixelmouse.es tiene caducado el certificado 8/8/2020. Contactados por twitter a ver si dan señales.

@tapia -> OpenShine se integra en otra empresa. ¿Seguís teniendo :snake: :snake: :snake:? En ese caso, ¿podéis abrir un PR con los datos de la nueva empresa?

@Xevib -> entiendo que Gisce continua su actividad (he visto que sigue teniendo commits en sus repos de GitHub)

@tanrax -> ¿sigue la actividad con pixelmouse?

**Comment by Xevib** [comment]
@quobit ya no trabajo alli pero por lo que se sigue en activo.

**Comment by tanrax** [comment]
@quobit continua, y más activo que nunca. Todo el back-end que se vende es Python.

**Comment by n3storm** [comment]
Año 2022:

Openshine dominio en venta

pixelmouse.es no responde nada

**Comment by quobit** [comment]
Openshine lo eliminamos ya que no contestaron. pixelmouse.es puede que haya hecho un 'rebranding' a .sapps preguntamos a @tanrax

**Comment by tanrax** [comment]
Exacto, ya no somos PixelMouse . Ahora nos llamamos Sapps con la web en https://sapps.studio

**Comment by quobit** [comment]
@tanrax ¿el resto de datos sigue igual? ¿o te has pasado a Clojure ;-) ?

**Comment by tanrax** [comment]
Jajaj seguimos y más que nunca 😄 Mañana reviso los datos
